### DISCUSSION TITLE
Bluetooth don't work

### DISCUSSION THREAD

**Discussion by Ultra1234567** [blueman-project/blueman#2351] [Q&A]
blueman:2.3.5
BlueZ:
Distribution: mint 21.3
Desktop environment: cinnamon

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

Kernel update and system completed in the latest version.
Bluetooth recently does not connect to the earphones.
 Try several devices.
 Probable bug on latest fix

**Comment by cschramm** [comment]
Most probably falls into the responsibility of your audio server. The default should be pulseaudio an Linux Mint 21.3.

What do you refer to with "latest fix"?
